Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -219,6 +219,13 @@
     --image-button-navigation-item-width: 26px;
 
     --blackboxed-opacity: 0.5;
+
+    --slider-thumb-background: white;
+    --slider-thumb-background-active: hsl(0, 0%, 94%);
+    --slider-thumb-box-shadow: 0px 0 2px 0 hsl(0, 0%, 55%);
+    --slider-track-background: hsl(0, 0%, 88%);
+    --slider-track-box-shadow: inset 0 0 3px hsl(0, 0%, 70%);
+    --slider-track-tick-background: hsl(0, 0%, 70%);
```

Comment:
Do we need to make any tweaks to the inactive style to match the inspiration in System Settings?

Reply:
Is there an inactive state for sliders in the System Settings?

The current styles follow as closely as possible the styles in System Settings except the tone of the grays which have a brown tint in the system panels. For Web Inspector I stuck with simple grays.

Reply:
I guess there isn't 😅